lib/internal/modules/esm/loader.js

```diff
@@ -273,26 +261,10 @@ class ModuleLoader {
     if (job !== undefined) {
       return job.module.getNamespaceSync();
     }
-
     // TODO(joyeecheung): refactor this so that we pre-parse in C++ and hit the
     // cache here, or use a carrier object to carry the compiled module script
     // into the constructor to ensure cache hit.
```

Comment:
Does this PR take care of this TODO?

Reply:
No because we don't preparse the module yet.